src/test/java/cz/czechitas/AdminShopPage.java

```diff
@@ -0,0 +1,51 @@
+package cz.czechitas;
+
+import java.util.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.*;
+import okhttp3.internal.http2.*;
+
+public class AdminShopPage {
+
+
+    WebDriver driver;
+   
+
+
+    public AdminShopPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+
+    public void openAdminpage() {
+        driver.navigate().to(Settings.adminLogonUrl);
+    }
+
+    public void fillLoginDetails (String username, String password) {
+        WebElement usernameField = driver.findElement(By.id("username"));
```

Comment:
Jeste bych zvazila ty By selectory vytahnout jako promenne na uroven tridy. Nutne to neni, ale je to trochu prehlednejsi.